bin/hdfcoinc/pycbc_multiifo_sngls_findtrigs

```diff
@@ -0,0 +1,292 @@
+#!/usr/bin/env python
+import h5py, argparse, logging, numpy, numpy.random
+from ligo.segments import infinity
+from pycbc.events import veto, coinc, stat
+import pycbc.conversions as conv
+import pycbc.version
+from numpy.random import seed, shuffle
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--verbose", action="count")
+parser.add_argument("--version", action="version", version=pycbc.version.git_verbose_msg)
+parser.add_argument("--veto-files", nargs='*', action='append', default=[],
+                    help="Optional veto file. Triggers within veto segments "
+                         "contained in the file are ignored")
+parser.add_argument("--segment-name", nargs='*', action='append', default=[],
+                    help="Optional, name of veto segment in veto file")
+parser.add_argument("--trigger-file",type=str,
+                    help="File containing single-detector triggers")
+parser.add_argument("--template-bank", required=True,
+                    help="Template bank file in HDF format")
+# produces a list of lists to allow multiple invocations and multiple args
+parser.add_argument("--statistic-files", nargs='*', action='append', default=[],
+                    help="Files containing ranking statistic info")
+parser.add_argument("--ranking-statistic", choices=stat.statistic_dict.keys(),
+                    default='newsnr',
+                    help="The single-detector ranking statistic to calculate")
+parser.add_argument("--statistic-keywords", nargs='*',
+                    default=[],
+                    help="Provide additional key-word arguments to be sent to "
+                         "the statistic class when it is initialized. Should "
+                         "be given in format --statistic-keywords "
+                         "KWARG1:VALUE1 KWARG2:VALUE2 KWARG3:VALUE3 ...")
+parser.add_argument("--loudest-keep-values", type=str, nargs='*',
```

Comment:
This is unused no?

Reply:
yes thats right - I thought I'd managed to remove all unused arguments must have missed that one